src/compiler/types.ts

```diff
@@ -3650,14 +3651,14 @@ export type AssertClause = ImportAttributes;
 
 export type ImportAttributeName = Identifier | StringLiteral;
 
-export interface ImportAttribute extends Node {
+export interface ImportAttribute extends Declaration {

```

Comment:
Making import attributes extend `Declaration` feels wrong; what was the thinking behind these?

Reply:
@jakebailey This is one of my questions, you beat me with your review ;). Is it okay to handle types with TransientSymbol (and them in createAnonymousType), especially in cases like completion? I was thinking about using contextual type to get a list of available attributes. Alternatively, is it okay to simply extend `getTypeAtLocation` to handle that case for services?